source/rdo/faq.html.md

```diff
@@ -13,7 +13,9 @@ wiki_last_updated: 2014-10-07
 
 ## What is RDO?
 
-RDO is two things. It's a freely-available, community-supported distribution of OpenStack that runs on Red Hat Enterprise Linux, CentOS, Fedora, and their derivatives. In addition to providing a set of software packages, it's also a community of users of cloud computing platform on Red Hat Linux operating systems to get help and compare notes on running OpenStack.
+RDO is two things. It's a freely-available, community-supported distribution of OpenStack that runs on Red Hat Enterprise Linux (RHEL), CentOS, Fedora, and their derivatives. In addition to providing a set of software packages, it's also a community of users of cloud computing platform on Red Hat-based operating systems to get help and compare notes on running OpenStack.
```

Comment:
@hguemar Should we just drop Fedora from the whole document or rephrase that we provide devel packages in trunk?  We do state that here:

https://www.rdoproject.org/rdo/release-cadence/

Thanks!